build.gradle

```diff
@@ -33,11 +33,11 @@ configurations {
 }
 
 dependencies {
-    shade 'fr.inria.gforge.spoon:spoon-core:5.7.0'
+    shade 'fr.inria.gforge.spoon:spoon-core:5.5.0'
     compile 'org.ow2.asm:asm:5.2'
     compile 'com.google.guava:guava:21.0'
     compileOnly 'com.google.errorprone:error_prone_annotations:2.0.15'
-    compileOnly 'com.google.code.findbugs:jsr305:3.0.1'
+    compile 'com.google.code.findbugs:jsr305:3.0.1'
```

Comment:
Was this change necessary?

Reply:
This cause issues in code as the code references the `@Nullable` or `@Nonnull` class when it checks for the annotations in source.

Reply:
@ryantheleach More exactly, these annotations are used at runtime here: https://github.com/SpongePowered/event-impl-gen/blob/0b10b0fac267f1809c002f983def01339729f7ac/src/main/java/org/spongepowered/eventimplgen/factory/ClassGenerator.java#L168-L174